pcm-kubernetes.yaml

```diff
@@ -0,0 +1,160 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: intel-pcm
+  labels: # uses host features by design privileges required
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/audit-version: latest
+    pod-security.kubernetes.io/warn: privileged
+    pod-security.kubernetes.io/warn-version: latest
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/instance: pcm
+    app.kubernetes.io/name: intel-pcm
+    app.kubernetes.io/part-of: intel-pcm
+  name: intel-pcm
+  namespace: intel-pcm
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: pcm-sensor-server
+      app.kubernetes.io/instance: pcm
+      app.kubernetes.io/name: intel-pcm
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: pcm-sensor-server
+        app.kubernetes.io/instance: pcm
+        app.kubernetes.io/name: intel-pcm
+        app.kubernetes.io/part-of: intel-pcm
+        jobLabel: pcm
+    spec:
+      automountServiceAccountToken: false
+      containers:
+      - image: ghcr.io/opcm/pcm:latest
+        env:
+        - name: PCM_IGNORE_ARCH_PERFMON
+          value: "0"
+        - name: PCM_NO_PERF
+          value: "0"
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 9738
+            scheme: HTTP
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        name: intel-pcm
+        ports:
+        - containerPort: 9738
+          hostPort: 9738
+          name: pcm-metrics
+          protocol: TCP
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 9738
+            scheme: HTTP
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        securityContext:
+          privileged: true
```

Comment:
Please drop this and add nessesary enviornment value 
PCM_NO_MSR (comment above)